src/common/style/base.css

```diff
@@ -64,47 +64,3 @@ a:focus {
 body {
   background-color: #FFF;
 }
-
-h1,
```

Comment:
These just get in the way of we're using react components for h1-6, also they're not in line with current Vanilla afaics.